README.md

```diff
@@ -1 +1,126 @@
-# to-do-list
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Awesome Books With ES6</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 To-Do List ](#-to-do-list-)
+  - [🛠 Built With ](#-built-with-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+  - [👥 Authors ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+- [🔭 Future Features ](#-future-features-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
```

Comment:
- [ ] 👍 Please consider updating your table of contents to align with the README template. The table of contents elements are not properly listed as per the template, especially the "Acknowledgements" and "License" section titles. Keeping the structure consistent with the template will make your README more organized and user-friendly.

| Your | Professional |
| --- | --- |
| ![image](https://github.com/noelfoka/to-do-list/assets/55840999/069d8db8-5bde-46e4-ba84-1d1777ec2f69) | ![image](https://github.com/noelfoka/to-do-list/assets/55840999/2fc820b6-68ff-4b97-a2aa-1690e33c92ca) |